platform/ios/app/MBXAppDelegate.m

```diff
@@ -13,7 +13,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 #ifndef MGL_DISABLE_LOGGING
     [MGLLoggingConfiguration sharedConfiguration].loggingLevel = MGLLoggingLevelFault;
 #endif
-
+    [[MGLOfflineStorage sharedOfflineStorage] setMaximumAmbientCacheSize:30000000 withCallback:^(NSError * _Nullable error) {
```

Comment:
Tip: when specifying big numbers that could be difficult to read or easily mistake for a different number, make them a named variable.

(Some languages allow you to insert underscores [e.g., in place of commas] to create visual separation... unfortunately, objc is not one of those languages.)